themes-default/slim/views/partials/submenu.mako

```diff
@@ -23,7 +23,7 @@
                 % endif
             % endfor
             <% curShowSlug = show.identifier.slug if show is not UNDEFINED else '' %>
-            <show-selector v-if="'${action}' === 'displayShow'" current-show-slug="${curShowSlug}"></show-selector>
+            <show-selector v-if="'${action}' === 'displayShow'" show-slug="${curShowSlug}"></show-selector>
```

Comment:
Maybe an idea to change `'${action}' === 'displayShow'` into a computed? Then we keep the logic from the template as much as possible.

Reply:
Where would we put the action though?
The submenu is a partial, I can't create a Vue instance here,
and the action variable is a global.

Reply:
aah didn't notice that was mako templated

Reply:
Could you move this to the vue submenu data field so this would be

<show-selector v-if="action === 'displayShow'" show-slug="${curShowSlug}"></show-selector>